frb_codegen/src/commands.rs

```diff
@@ -208,9 +235,16 @@ fn ffigen(
     std::io::Write::write_all(&mut config_file, config.as_bytes())?;
     debug!("ffigen config_file: {:?}", config_file);
 
-    // NOTE please install ffigen globally first: `dart pub global activate ffigen`
+    let cmd = if guess_toolchain(dart_root).unwrap() == DartToolchain::Dart {
+        "dart run"
+    } else {
+        "flutter pub run"
+    };
```

Comment:
Seems that we already have a new method for this: `DartToolchain::as_run_command`

Reply:
done in 94c11a4e8e7d0dd6c0671f7f319379f66cfb3d79